### DISCUSSION TITLE
How to disable multiple selection in a Dropzone ?

### DISCUSSION THREAD

**Discussion by Sakkhor909** [mantinedev/mantine#835] [Q&A] [answered]
When I select multiple images and in a dropzone, the onDrop function invokes but is there a way that instead of this, onReject invoke?

**Comment by rtivital** [comment, marked as answer]
There is no way to disable it, instead you can just process the first dropped file and discard the rest

**Reply by Sakkhor909** [reply]
Thank you much for your answer. I think it would be a nice feature to have. Also, Is there a way enable `required` it inside a form?
